### PR TITLE
add spectragram@1.0.3 w/ git autoupdate.

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2,
Pull request for issue: #9547 
Please help me for reviewing, thank you all!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/adrianengine/jquery-spectragram
 * Official website (optional, not the repository): http://spectragram.js.org/
 * NPM package url (optional): https://www.npmjs.com/package/spectragram
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 336
   - Count of watchers: 30
   - Count of forks: 119
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 25
   - Downloads in the last month: 139

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/spectragram/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "jquery-plugin",
+    "ecosystem:jquery",
+    "gallery",
+    "instagram",
+    "photos",
+    "spectragram"
+  ],
+  "author": "Adrian Quevedo <npm@adrianquevedo.com> (http://adrianquevedo.com/)",
+  "license": "MIT",
+  "homepage": "https://spectragram.js.org/",
```
Comment (pvnr0082t) [inline ajax/libs/spectragram/package.json]: I think it doesn't support SSL. We can just use `http` here.

File: ajax/libs/spectragram/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "npmName": "spectragram",
```
Comment (pvnr0082t) [inline ajax/libs/spectragram/package.json]: There is officially minified file on GitHub for each version but npm not. Let's use git auto-update config here. Thanks.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
I have the commit corrected, and please help me for reviewing, thank you all!

**Comment by maruilian11** [comment]
have you opened the issue?

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Thanks guys.

**Pull request merged**